src/test/kotlin/com/codingchallenge/transactionanalyser/TransactionAnalyserAppTest.kt

```diff
@@ -0,0 +1,41 @@
+package com.codingchallenge.transactionanalyser
```

Comment:
You mentioned on Slack that you added tests after. 
I want you to start getting in the mindset that tests drive the design of your code - that is, the practice of TDD. Test Driven Development. It is not an answer to everything I know, but in my view it will change your view on how you bring new features in and have tests that cover them. 
Many years ago Guy asked me to view the following video by the man himself uncle bob: https://vimeo.com/43734265 yes he is awkward but pay attention to what he is trying to teach and what it achieves by writing tests before your implementation. 
Also see: https://blog.cleancoder.com/uncle-bob/2014/12/17/TheCyclesOfTDD.html
I want you to try this when we are pairing together next.

Reply:
Will definitely look into TDD.